opta/layer.py

```diff
@@ -222,7 +222,7 @@ def load_from_yaml(
             raise UserErrors(f"File {config} not found")
 
         conf["original_spec"] = config_string
-        conf["path"] = config
+        conf["path"] = config_path
```

Comment:
This sounds like a series issue.. was this broken before?

Reply:
great question, it could have been, but this field was rarely ever used in the regular layer, and never used by the parent (where the git redirect issue was present)